src/components/pages/search.js

```diff
@@ -96,33 +151,49 @@ function Search({ PageTemplate }) {
   return (
     <PageTemplate alerts={alerts}>
       <div className="SearchPage">
-        <div className="SearchPage-searchHeader">
+        <div className="SearchPage-headline">
+          <h1>{t('headline', { count: displayedDocs.length })}</h1>
+        </div>
+        <div className="SearchPage-controls">
           <SearchBar initialValue={searchText} onSearch={setSearchText} />
+          <Select
+            showArrow
+            showSearch
+            size="large"
+            value={null}
+            options={tagOptions}
+            onChange={handleSelectTag}
+            placeholder={(
+              <span className="SearchPage-filterPlaceholder">
+                <span>{`${t('filterPlaceholder')} (${selectedTags.length})`}</span>
+                <SearchOutlined className="SearchPage-filterPlaceholderIcon" />
+              </span>
+            )}
+            suffixIcon={<MenuOutlined />}
```

Comment:
Once Mani provides the proper icon, we can replace this one.
Sadly we can't get rid of the search icon before it, because it goes out of the box with the option to search within the options list. But I think we can live with it, it makes it more intuitive actually.

Reply:
Changed!